cmd/restart.go

```diff
@@ -31,7 +31,7 @@ func Restart() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "restart",
-		Short: "Restarts the pods of your development environment",
+		Short: "Restarts the services of your okteto manifest",
```

Comment:
This is the only one that sounds odd. The rest of the changes are much more clear now.
What about `Restarts the pods containing your development container`? or `Restart the pod of your development container` or something like that... I'm not sure if my proposal is accurate enough.

Reply:
This command restarts the pods indicated by the `services` field of the okteto manifest. It does not restart the dev container. What about something like:
`Restarts the pods references by the "services" field of the okteto manifest`

Reply:
Sounds more accurate like that then

Reply:
In kubernetes you don't restart services, you recreate (or restart) the pods. I think we should use more  kubernetes-specific in this kind of descriptions.

Reply:
Maybe: restarts the deployment listed in the `services` field of the okteto manifest?